### ISSUE TITLE
Feature Request: Multiple exclude and searchCadence behavior based on timestamp

### ISSUE THREAD

**Issue by Byter09** [cross-seed/cross-seed#834]
### Does your feature request solve a problem you're facing?

When setting up `excludeOlder` and `excludeRecentSearch` I'm faced with the predicament that when a new release comes out I want to quickly gather all the cross seeds from all the various trackers I have, so that I can be part of the initial seed for all of them. However, after some time you may not want that anymore, because at that point it is unlikely that trackers will add the release, so there's no point in searching.

### Does an existing issue/feature request already exist?

- [X] I've searched far and wide!

### What would you like for cross-seed to do?

There's two things cross-seed could do:
1. Set up two new variables. One decides when this threshold happens, and the other one is just the same as `excludeRecentSearch` but for whatever that new time period needs. Say for example we define a "recent search" and a "longterm search". `excludeRecentSearch` could then be used to define how often a torrent is searched for while in the recent search period. Then, using said other variable like `longtermThreshold`, if a time threshold has been reached, cross-seed switches the torrent into longterm search mode, where it will use another variable like `excludeLongtermSearch`. So for new additions to the download client a fast search is used, and when the threshold has been reached a slower time between searches is used for it.
2. This version is a bit more complicated but probably slightly better: Temporal Decay. Basically all that needs to be defined is a search period, which we already have thanks to `excludeOlder`. Initially the search will be very fast (say once an hour), and then the closer the age of the source torrent becomes to `excludeOlder` the less often it will be searched for, ending in a final search once `excludeOlder` time period has been reached.

### What are you currently doing, if anything, to solve or work around the issue?

I have set my `excludeRecentSearch` to a very small value (currently 3 hours), and balance my `excludeOlder` so that not too many torrents are searched for. As you can imagine this requires adjustments the more torrents one has.

### Additional comments

I'm unsure which of the two proposed solutions is the best for the "normal usecase". So maybe there's even another solution that fits more in with what cross-seed already does. I also wanted to ask on the discord first but my message blew up so big that I figured a full feature request is better.

I also hope I have sufficiently explained what I mean. I struggle with that :)

**Comment by Byter09** [comment]
It might be that I've simply never seen RSS working. If that's already doing what it's supposed to do then I guess I'm good.

What I could try and do is set both variables to recommended values and check whether I still join early and initial seeding. Thanks for letting me know!

You can close this issue if you think it's a useless feature request, but I'll see if I can update on my issue in about a week and whether everything works as you say it should.

**Comment by Byter09** [comment]
Closing this as I just discovered that I was using `cross-seed search` in a loop. This either must've been a remnant of debugging or the compose is so old that back then the daemon didn't even exist.

Changed it to the recommended startup method, was screamed at by misconfigured config options, fixed those, and now have the daemon running.

Thanks for getting back to me so fast. Would've never found this as I rarely change compose files haha :D
